File: common/src/main/java/com/tnt_man_inc/jarm_3/Registries.java
package com.tnt_man_inc.jarm_3;

import me.shedaniel.architectury.registry.DeferredRegister;
import net.minecraft.core.Registry;
import net.minecraft.world.item.Item;
import net.minecraft.world.level.block.Block;

public class Registries {
    public static final String MOD_ID = "jarm_3";
    public static final DeferredRegister<Item> ITEMS = DeferredRegister.create(MOD_ID, Registry.ITEM_REGISTRY);
    public static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(MOD_ID, Registry.BLOCK_REGISTRY);
}

File: common/src/main/kotlin/com/tnt_man_inc/jarm_3/Registries.kt
package com.tnt_man_inc.jarm_3

import com.tnt_man_inc.jarm_3.items.NISGDApple
import me.shedaniel.architectury.registry.DeferredRegister
import net.minecraft.core.Registry
import java.lang.AssertionError

object Registries {
    const val MOD_ID = "jarm_3"
    val ITEMS = DeferredRegister.create(MOD_ID, Registry.ITEM_REGISTRY)
    val BLOCKS = DeferredRegister.create(MOD_ID, Registry.BLOCK_REGISTRY)
}